pkg/operator/ceph/controller/spec.go

```diff
@@ -653,8 +655,28 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 	return sec
 }
 
-// LogCollectorContainer runs a cron job to rotate logs
+// LogCollectorContainer rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	maxLogSize := resource.MustParse("500M")
```

Comment:
Actually, an even simpler suggestion is to just use a string directly for maxLogSize, then we can avoid MustParse in both places.
```suggestion
	maxLogSize := "500M"
```

Reply:
not required anymore